pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -285,7 +285,7 @@ func (r *ReconcileComplianceScan) validate(instance *compv1alpha1.ComplianceScan
 func (r *ReconcileComplianceScan) phasePendingHandler(instance *compv1alpha1.ComplianceScan, logger logr.Logger) (reconcile.Result, error) {
 	logger.Info("Phase: Pending")
 	// Remove annotation if needed
-	if instance.NeedsRescan() {
+	if instance.NeedsRescan() || instance.NeedsTimeoutRescan() {
```

Comment:
Should be able to add a test for this in https://github.com/ComplianceAsCode/compliance-operator/blob/master/pkg/controller/compliancescan/compliancescan_controller_test.go